doc/nrf/protocols/matter/getting_started/adding_clusters.rst

```diff
@@ -94,17 +94,90 @@ This tool is provided with the Matter repository in the |NCS|.
 
 To edit clusters using the ZAP tool, complete the following steps:
 
-1. Make sure that you have installed the ZAP tool prerequisites, as mentioned in the Quick instructions section in the :file:`README.md` file in the official `ZCL Advanced Platform`_ repository.
-#. Open your installation directory for the |NCS| in a command line.
-#. Navigate to :file:`modules/lib/matter`.
+1. Download the ZAP package containing pre-compiled executables and libraries, extract it, and add its location to the system :envvar:`PATH` environment variables:
+
+   .. note::
+       You can download the package in a compatible version manually from the `ZCL Advanced Platform releases`_, but it's recommended to use a dedicated helper script that will do it by yourself.
+
+   a. Open your installation directory for the |NCS| in a command line.
+   #. Navigate to :file:`modules/lib/matter`.
+   #. Run the helper script to download and extract a ZAP package in the location given by the ``location_path`` argument:
+
+      .. code-block::
+
+         python3 scripts/setup/nrfconnect/get_zap.py -l location_path
```

Comment:
Does not work under Toolchain Manager on Windows, though this works:

```suggestion
         python scripts/setup/nrfconnect/get_zap.py -l location_path
```

Reply:
Script itself has three issues:

1) Incorrect URL for windows:
`https://github.com/project-chip/zap/releases/download/v2022.12.20-nightly/zap-windows.zip` -> `https://github.com/project-chip/zap/releases/download/v2022.12.20-nightly/zap-win.zip`.

2) Incorrect executable file:
`zap` -> `zap.exe` (possible the same issue with `zap-cli` later on.

3) `wget` is not part of the Toolchain Manager. I suggest to add it to requirements of SDK and ensure it will be a part of the next Toolchain Manager for NCS 2.3.0. Alternative is to add the documentation here (not preferred).

Reply:
Issues number 1 and 2 were fixed on the corresponding PR to sdk-connectedhomeip. Regarding issue number 3, I added `wget` to NCS requirements in this PR.